frontend/src/components/shared/StreamlitMarkdown/styled-components.ts

```diff
@@ -25,15 +26,20 @@ function convertRemToEm(s: string): string {
   return s.replace(/rem$/, "em")
 }
 
+function sharedMarkdownStyle(theme: Theme): any {
+  return {
+    a: {
+      color: theme.colors.linkText,
+    },
```

Comment:
Just an opinion, but I think it is more confusing to have this `sharedMarkdownStyle` slice separated out within the same file vs. just adding a tag styling to both `StyledStreamlitMarkdown` & `StyledHeaderContainer`. Perhaps because I can't imagine there would be significantly more styling properties added that would be shared across those components.